include/klee/Internal/ADT/DiscretePDF.inc

```diff
@@ -162,32 +162,32 @@ void DiscretePDF<T>::update(T item, weight_type weight) {
 
 template <class T>
 T DiscretePDF<T>::choose(double p) {
-  if (p<0.0 || p>=1.0) {
+  if ((p < 0.0) || (p >= 1.0))
     assert(0 && "choose: argument(p) outside valid range");
-  } else if (!m_root) {
+
+  if (!m_root)
     assert(0 && "choose: choose() called on empty tree");
```

Comment:
I'm not sure about this, as `llvm_unreachable` has a different semantics from `assert(0)` in release mode.  (Also note that this was old behavior of the function, I haven't changed it.)